packages/network/src/network.js

```diff
@@ -14,6 +15,125 @@ import '@agoric/store/exported.js';
  */
 export const ENDPOINT_SEPARATOR = '/';
 
+const Shape1 = /** @type {const} */ ({
```

Comment:
I'm wondering if we can have something more descriptive here from `Shape1`/`Shape2`?

Reply:
any chance you're more inspired than I am? got a suggestion?